### ISSUE TITLE
Report figures do not include komplexity-filtered reads

### ISSUE THREAD

**Issue by louiejtaylor** [sunbeam-labs/sbx_report#12]
The komplexity-filtered reads are not shown in the QC overview stacked barplot. This may be at heart a QC issue, as they are also not included in the preprocess_summary.tsv table. Thoughts on whether I should open a main sunbeam issue for this, @ressy? (Both of these should hopefully be a quick fix).

Figured this out in a dataset where 80-90% of the reads were low-complexity and I was wondering where the heck they all went.

**Comment by ressy** [comment]
Yeah if we should change that table creation in sunbeamlib.reports.summarize_qual_decontam (does that look like the right place?) then we should have a sunbeam issue for it, I think.  Thanks!
